packages/host/app/components/operator-mode/code-submode/playground-panel.gts

```diff
@@ -484,33 +493,45 @@ class PlaygroundPanelContent extends Component<PlaygroundContentSignature> {
     </style>
   </template>
 
-  fieldFormats: Format[] = ['embedded', 'fitted', 'atom', 'edit'];
+  @consume(GetCardContextName) private declare getCard: getCard;
+
   @service private declare cardService: CardService;
   @service private declare loaderService: LoaderService;
   @service private declare operatorModeStateService: OperatorModeStateService;
   @service private declare realm: RealmService;
   @service private declare realmServer: RealmServerService;
   @service private declare recentFilesService: RecentFilesService;
   @service private declare playgroundPanelService: PlaygroundPanelService;
-  @tracked newCardJSON: LooseSingleCardDocument | undefined;
-  @tracked fieldChooserIsOpen = false;
+  @tracked private newCardJSON: LooseSingleCardDocument | undefined;
+  @tracked private fieldChooserIsOpen = false;
+  @tracked private cardResource: ReturnType<getCard> | undefined;
```

Comment:
So this is @tracked becos we are dynamically providing the context? is that correct?

Reply:
no. it's `@tracked` because the resource is not set in the constructor. so we need to tell glimmer that it needs to pay attention to this property when it's consumed as it will no longer be undefined as soon as the DOM renders